File: atinject-api/src/main/java/org/atinject/api/user/dto/GetUserRequest.java
package org.atinject.api.user.dto;

import java.util.UUID;

import org.atinject.core.websocket.dto.WebSocketRequest;

public class GetUserRequest extends WebSocketRequest {

    private static final long serialVersionUID = 1L;
    
    private UUID userId;

    public UUID getUserId() {
        return userId;
    }

    public GetUserRequest setUserId(UUID userId) {
        this.userId = userId;
        return this;
    }
    
    
}

File: atinject-integration/src/main/resources/webapp/scripts/atinject/user/GetUserRequest.js
define(["atinject/WebSocketRequest"],
function(WebSocketRequest){
    function GetUserRequest() {
        this._class = "org.atinject.api.user.dto.GetUserRequest"
    };

    GetUserRequest.prototype = new WebSocketRequest();

        GetUserRequest.prototype.getUserId = function() {
        return this.userId;
    };

    GetUserRequest.prototype.setUserId = function(userId) {
        this.userId = userId;
        return this;
    };


    
    return GetUserRequest;
});
